### DISCUSSION TITLE
grouping Columns and sub Row component at the same time

### DISCUSSION THREAD

**Discussion by SalvadorLucas** [TanStack/table#3294] [Q&A]
How can I use grouping columns and row sub component at the same time?
At this moment both are using the same row.isExpanded flag to display subComponent and Aggegated data, so when I try to visualize grouped column's data the first row Sub component is displayed too, that's happend on every first row.
Thanks for helping.

**Comment by frankbolviken** [comment]
Are there any suggested solutions for this?

**Comment by davidcheal** [comment]
Took me ages to work this out, but this can be done by checking the subrow count.

{row.getIsExpanded() && row.subRows.length === 0 && (
<content>
)
